docs/content/docs/dev/table/tableApi.md

```diff
@@ -1454,21 +1454,23 @@ result3 = table.order_by(table.a.asc).offset(10).fetch(5)
 
 {{< label Batch >}} {{< label Streaming >}}
 
-Similar to the `INSERT INTO` clause in a SQL query, the method performs an insertion into a registered output table. The `executeInsert()` method will immediately submit a Flink job which execute the insert operation.
+Similar to the `INSERT INTO` clause in a SQL query, the method performs an insertion into a registered output table. 
+The `insertInto()` method will translate the `INSERT INTO` to a `TablePipeline`. 
```

Comment:
maybe `convert` is better than `translate`?

Reply:
I understand, I also don't think `convert` is 100% matching the context as well.
`transform` maybe?